### PR TITLE
fix: DASH-410

### PR DESCRIPTION
**Pull request by jnsdls**
FIXES: DASH-410

---

## PR-Codex overview
This PR focuses on improving error handling and code clarity in the dashboard application. It modifies the `package.json` scripts and enhances the JWT handling in the `thirdweb.client.ts` file, along with a minor adjustment in type handling in the `hooks.ts` file.

### Detailed summary
- Removed `--turbopack` option from the `dev` script in `package.json`.
- Added error handling for missing JWT in `thirdweb.client.ts`.
- Updated the return statement to ensure a JWT is always returned.
- Adjusted the return type in `hooks.ts` to ensure values are correctly handled as `BigInt`.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by jnsdls** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/5328?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#5328** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/5328?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#5318** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/5318?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @jnsdls and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/[contractAddress]/_components/claim-conditions/claim-conditions-form/hooks.ts [hunk 1]
```diff
@@ -208,6 +208,7 @@ function toBigInt(value: string | number | undefined): bigint | undefined {
   if (value === "unlimited") {
     return maxUint256;
   }
+  return BigInt(value);
```
Comment (jnsdls) [inline apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/[contractAddress]/_components/claim-conditions/claim-conditions-form/hooks.ts:211]: this is the actual fix... `toBigInt` was never returning the actual... bigint

**Review by MananTank — APPROVED** [review]

File: apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/[contractAddress]/_components/claim-conditions/claim-conditions-form/hooks.ts (hunk 1 above)
Comment (joaquim-verges) [reply apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/[contractAddress]/_components/claim-conditions/claim-conditions-form/hooks.ts:211]: cc @kien-ngo for your other PR that is related i think?

Comment (kien-ngo) [reply apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/[contractAddress]/_components/claim-conditions/claim-conditions-form/hooks.ts:211]: ugrr I accidentally moved that [return BigInt] to another function (for the erc20 fix) - mb!!

**Pull request merged**